internal/storage/bucket_handle.go

```diff
@@ -334,6 +334,12 @@ func (bh *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsReq
 		IncludeFoldersAsPrefixes: req.IncludeFoldersAsPrefixes,
 		//MaxResults: , (Field not present in storage.Query of Go Storage Library but present in ListObjectsQuery in Jacobsa code.)
 	}
+	err = query.SetAttrSelection([]string{"Name", "Size", "Generation", "Metageneration", "Updated", "Metadata", "ContentEncoding", "CRC32C"})
```

Comment:
I see this method is returning gcs.Object but gcs.MinObject is inserted into cache.
Do we want to change the callers of this method to use gcs.MinObject directly? You can do that in a followup cl.

Reply:
Doing this in this [PR](https://github.com/GoogleCloudPlatform/gcsfuse/pull/2657)